go.mod

```diff
@@ -1,3 +1,3 @@
 module github.com/madlambda/spells
 
-go 1.11
+go 1.16
```

Comment:
The ability to do this does feel like an improvement, like make it explicit we need at least Go 1.16 + get a clear error if someone is using a previous version (instead of a build error, that may not be very clear every time).

Reply:
yep